docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -277,6 +277,25 @@ Setting a combination of properties to both audio and subtitle tracks.
 
 Support is confirmed for `track_description`, `track_role` and `track_name` properties, but other properties may be supported.
 
+## FILE PROPERTIES
+
+File properties can be used to set the track kind and language in audio tracks. Track kind allows accessibility options, such as audio description tracks, to be indicated. The language of the track can also be changed using this property. This replaces the legacy behaviour of specifying the language between the last underscore and the extension (`_<language>.m4a`).
+
+Here is the list of [available track kinds](https://html.spec.whatwg.org/multipage/media.html#dom-audiotrack-kind).
+
+Marking audio1.m4a as the Dutch audio description track:
```

Comment:
Could perhaps benefit from being more of a sentence, such as "Here is an example of how to..."